packages/@aws-cdk/aws-lambda-nodejs/lib/bundling.ts

```diff
@@ -140,6 +142,10 @@ export class Bundling implements cdk.BundlingOptions {
       ...loaders.map(([ext, name]) => `--loader:${ext}=${name}`),
       ...this.props.logLevel ? [`--log-level=${this.props.logLevel}`] : [],
       ...this.props.keepNames ? ['--keep-names'] : [],
+      ...this.props.tsconfig ? [`--tsconfig=${pathJoin(inputDir, this.relativeTsconfigPath!)}`] : [],
```

Comment:
would rewrite this to avoid the non null assertion (and define `this.relativeTsconfigPath` above only if we have `props.tsconfig`)

```suggestion
      ...this.relativeTsconfigPath ? [`--tsconfig=${pathJoin(inputDir, this.relativeTsconfigPath)}`] : [],
```

Reply:
Done